Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -8,6 +8,7 @@ import Shared
 import XCGLogger
 
 private let log = Logger.browserLogger
+private let desktopBookmarksLabel = NSLocalizedString("Desktop Bookmarks", tableName: "BookmarkPanel", comment: "The folder name for the virtual folder that contains all desktop bookmarks.")
```

Comment:
Nit: we used to capitalize the first letter of all file-level constants like this. Seeing `log` above, we might have dropped that convention somewhere along the way, though I'm not sure it was intentional.